primefaces/src/main/resources/META-INF/resources/primefaces/locales/locale-es.js

```diff
@@ -116,26 +116,26 @@ if (window.PrimeFaces) {
     day: "Día",
     list: "Agenda",
     allDayText: "Todo el día",
-    moreLinkText: "Más....",
+    moreLinkText: "Más...",
     noEventsText: "No hay eventos",
     aria: {
-      "datatable.sort.ASC": "ordenar columna ascendente",
-      "datatable.sort.DESC": "ordenar columna descendente",
-      "datatable.sort.NONE": "eliminar la clasificación en la columna",
+      "datatable.sort.ASC": "ordenar columna ascendentemente",
```

Comment:
"Ordenar columna de forma ascendente", IMO.

Reply:
Maybe, but I haven't seen any Spanish website with ''ascendentemente''. Is more clear ''... de forma ascendente''

Reply:
Yes, it's rigth.